### PR TITLE
feat: add internal padding to icons

### LINKED ISSUE #365: Document custom icon process and update hardcoded icons
- [x] Add icon contribution to our repo's docs, which is roughly:
- [x] If you need an icon or pictogram not offered through Carbon's packages, add it to this sketch file https://ibm.ent.box.com/folder/158366227035, in doing so, include attribution where required so we can do an open-source audit for all icons that didn't come from us
- [x] Size it at the intended size that it'll get rendered (might as well make sure it renders on the pixel grid for 1x displays if possible)
- [x] Ensure it's all black
- [x] Use a slice to export the SVG
- [x] Place in the icons package in monorepo
- [x] Run a build, then you'll be able to use the *optimized* SVG as a React component in the web app
- [x] Update the footer component with IBM SVG coming through our icons package (that doesn't get published to npm)

### PR DESCRIPTION
**Pull request by mattrosno**
Closes #365

#### Changelog

**New**

- SVG contribution docs

**Changed**

- Adds internal padding to custom SVGs

**Removed**

- NA

#### Testing / reviewing

These icons all look the same size now (before, GitHub and npm were visually bigger.)

image

And the framework icons still look good, now that they have the internal padding too.

image

### CODE REVIEW

**Review by alisonjoseph — APPROVED** [review]

File: docs/packages-icons.md [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+
+1. Download the latest [Sketch file from Box](https://ibm.ent.box.com/folder/158366227035).
+1. Add attribution where required so we can do manual open-source audits for all icons that didn't
+   come from Carbon icon and pictogram code packages.
+1. Size the SVG at its intended size so we can be pixel-precise for standard density displays (SVGs
+   can get resized in the web app so this isn't overly important.)
+1. Include an internal 2px padding for 32px icons per
+   [IBM Design Language specifications](https://www.ibm.com/design/language/iconography/ui-icons/design/#padding)
+   (1px padding for 16px icons.)
+1. Ensure the fill colors are all black.
```
Comment (alisonjoseph) [inline docs/packages-icons.md:26]: Are we not storing icons with colors in this package?
See https://next.carbondesignsystem.com/developing for example
Screen Shot 2022-06-16 at 10 07 51 AM

Comment (mattrosno) [reply docs/packages-icons.md:26]: Right now we just have black SVGs so they can take a single color (e.g. in catalog items.)

If we need colored SVGs that need multiple colors, we'll need to add those as separate SVGs.

image

Comment (mattrosno) [reply docs/packages-icons.md:26]: @alisonjoseph oh you're talking about all these? Maybe later we can do cleanup so these other icons (that got ported over from carbon-website) are in our Sketch source and properly attributed like the others.

image

**Review by andreancardona — APPROVED** [review]
Looks good!

**Pull request merged**